### DISCUSSION TITLE
Trivy not detecting vulnerabilities detected by other scanning tools

### DISCUSSION THREAD

**Discussion by faizu6** [aquasecurity/trivy#5977] [Q&A] [answered]
### Question

I am using Trivy from past 7 months in the company for scanning the vulnerable maven package version in the remote code repositories. Recently a person from engg. team pointed out that customer had found a vulnerability in our maven package repository which we were unable to detect. Upon further evaluation i found a vulnerability *CVE-2023-35116* but trivy was unable to find it in the same code base/repository. The tool customer used to scan was PRISMA. Then we tried OWASP Dependency check tool and the tool was able to find the same vulnerability but Trivy was not. Could anyone please provide me a reason for this? Do I need to switch to other tools for detecting the vulnerable package version in the maven repository.

### Target

Git Repository

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Operating System

ubuntu 22.04

### Version

```bash
$ trivy --version
Version: 0.48.3
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-12-06 06:11:28.706865382 +0000 UTC
  NextUpdate: 2023-12-06 12:11:28.706865051 +0000 UTC
  DownloadedAt: 2023-12-06 10:19:16.141587499 +0000 UTC
Policy Bundle:
  Digest: sha256:1df8ade71efc830877ca3b1130f83e0c6368e3a45b0d4c0f0418955501644054
  DownloadedAt: 2023-11-27 14:00:04.055075466 +0000 UTC
```

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @faizu6 
Thanks for your interest to Trivy!

We use `GitHub advisory database` for java advisories - https://aquasecurity.github.io/trivy/v0.48/docs/scanner/vulnerability/#data-sources_1
Unfortunately [CVE-2023-35116](https://github.com/advisories/GHSA-gx6w-fqg7-mc3p) is still `Unreviewed` - https://github.com/advisories/GHSA-gx6w-fqg7-mc3p.
If you have information about vulnerable package and affected versions - please suggest these changes in GitHub - https://github.com/advisories/GHSA-gx6w-fqg7-mc3p/improve.
This will speed up review this CVE.

Regards, Dmitriy
